dev/Code/Framework/AzGameFramework/AzGameFramework/Application/ServerApplication.h

```diff
@@ -0,0 +1,42 @@
+/*
+* All or portions of this file Copyright (c) Amazon.com, Inc. or its affiliates or
+* its licensors.
+*
+* For complete copyright and license terms please see the LICENSE at the root of this
+* distribution (the "License"). All use of this software is governed by the License,
+* or, if provided, by the license below or the license accompanying this file. Do not
+* remove or modify any license notices. This file is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*
+*/
+
+#ifndef AZGAMEFRAMEWORK_SERVERAPPLICATION_H
+#define AZGAMEFRAMEWORK_SERVERAPPLICATION_H
+
+#include <AzCore/base.h>
+#include "GameApplication.h"
+
+#pragma once
+
```

Comment:
We can replace old c style include guard with pragma once and put it before includes